build_tools/github_actions/configure_ci.py

```diff
@@ -57,9 +57,8 @@ def skip_path(path: str) -> bool:
 
 def set_output(d: Mapping[str, str]):
   print(f"Setting outputs: {d}")
-  step_output_file = os.environ["GITHUB_OUTPUT"]
-  with open(step_output_file, "a") as f:
-    f.writelines(f"{k}={v}" "\n" for k, v in d.items())
+  for k, v in d.items():
```

Comment:
Please don't undo this :-)

Reply:
Oh, files got messed up when testing the linter...